tests/onnxruntime/test_optimization.py

```diff
@@ -234,6 +234,10 @@ def preprocess_function(examples, tokenizer):
             self.assertEqual(expected_quantized_matmul, num_quantized_matmul)
             gc.collect()
 
+    def test_static_quantization_without_shape_inference_raises(self):
+        with self.assertRaises(expected_exception=ValueError):
+            QuantizationConfig(is_static=True, disable_shape_inference=True, format=QuantFormat.QDQ)
```

Comment:
Seems to be the correct place to put this if we insist on adding a test for `__pos_init__`, pinging @echarlaix for confirmation.

Reply:
you let me know!